tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -649,6 +662,24 @@ using OpConverter =
 
 class Converter {
  public:
+  // During conversion, this table is populated with quantization scales per tensor
+  std::unordered_map<nvinfer1::ITensor*, float> quantization_scales_;
```

Comment:
Please make these private members and add accessors and/or mutators accordingly.